src/utils.jl

```diff
@@ -308,10 +308,16 @@ end
 
 batchindex(xs, i) = (reverse(Base.tail(reverse(axes(xs))))..., i)
 
-function batch(xs::AbstractArray{<:AbstractArray{T,N}}) where {T, N}
-    sz = size(xs)
-    y = stack(vec(xs), dims=N+1)
-    reshape(y, size(y)[1:N]..., sz...)
+function batch(xs::AbstractArray{<:AbstractArray})
```

Comment:
might be worth leaveing a comment to https://github.com/JuliaLang/julia/pull/43334

Reply:
I guess that one day `batch` will just fall back to `Base.stack`